README.md

```diff
@@ -1,12 +1,16 @@
-# RHODS-Jira-Tools
+# RHODS-JIRA-Tools
 
-A collection of tools for managing Jira issues for the RHODS project
+A collection of tools for managing JIRA issues for the RHODS project
```

Comment:
This is very much a nit, but I think capitalizing Jira is actually incorrect. It's consistently written as "Jira" on Atlassian's website: https://www.atlassian.com/software/jira